manifests/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -180,6 +180,43 @@ data:
       controlPlaneAuthPolicy: NONE
       discoveryAddress: istiod.istio-system.svc:15012
 
+  Corefile: |-
+    global:15054 {
+        errors
+        log
+        proxy . 127.0.0.1:15053 {
+        }
+
+    }
+
+    .:15054 {
+        errors
+        log
+        health :15056 {
+          lameduck 5s
+        }
+
+        #proxy global 127.0.0.1:15010 {
+        #  protocol grpc insecure
+        #}
+        proxy global 127.0.0.1:15053 {
+        }
+        proxy svc 127.0.0.1:15053 {
+        }
+        proxy example.com 127.0.0.1:15053 {
```

Comment:
Sorry, since this PR appears to be in bike-shedding limbo I'm doing some tests, tricky to stash/unstash on each iteration. 

Will clean up when it gets close to approval.